.github/workflows/pages.yaml

```diff
@@ -3,7 +3,7 @@ name: Deploy to Pages
 # Runs on pushes targeting things that look like the default branch
 on:
   push:
-    branches: ["main", "master"]
+    branches: ["main", "master", "93-repo-reorg"]
```

Comment:
@bsaul  (or someone) - remove this branch from the github action when the PR is finally ready. Adding it here, as we'll want to test the pages.